model/src/main/kotlin/config/Excludes.kt

```diff
@@ -36,4 +41,32 @@ data class Excludes(
          */
         @JsonInclude(JsonInclude.Include.NON_EMPTY)
         val scopes: List<ScopeExclude> = emptyList()
-)
+) {
+    /**
+     * Return the [ProjectExclude] for this [project], or null if there is none.
+     */
+    fun findProjectExclude(project: Project) = projects.find { it.path == project.definitionFilePath }
```

Comment:
(Unfortunately) there could be multiple ProjectExcludes for the same Project, right? So this should `filter` to return a list, or?

Reply:
Anyway, maybe inline this to `isScopeExcluded()`?

Reply:
* Yes, technically there could, but this shouldn't be allowed. Once we implement validation for the config this should be reported as an error.
* The function will be needed in other places in upcoming commits.

Reply:
> Once we implement validation for the config this should be reported as an error.

Or maybe be merged upon deserialization? Anyway, that's to be determined later then.